example/app.py

```diff
@@ -19,17 +18,27 @@ async def realtime(payload):
     print("async realtime ", payload)
 
 if __name__ == "__main__":
-    ID = os.getenv("SUPABASE_ID")
+    ID = 'unjmcdnawyleplexbxja'#os.getenv("SUPABASE_ID")
```

Comment:
Do not commit personal credentials and IDs.

Reply:
This is a test project, but point taken.